<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoggingGUI</class>
 <widget class="QWidget" name="JoggingGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_y">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_z">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_a">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_b">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_c">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_offset">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_moveObject">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Obstacle</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_move</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_moveRobot">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Robot</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_move</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_move"/>
 </buttongroups>
</ui>
